storage/container_storage_interface/persistent-storage-csi-ebs.adoc

```diff
@@ -29,7 +29,8 @@ include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
 
 [IMPORTANT]
 ====
-{product-title} defaults to using an in-tree, or non-CSI, driver to provision AWS EBS storage. This in-tree driver will be removed in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree driver are planned for migration to the CSI driver at that time.
+{product-title} defaults to using an in-tree, or non-CSI, driver to provision AWS EBS storage. For {product-title}, automatic migration from AWS EBS in-tree to the CSI driver is available as a Technology Preview (TP) feature.
+With migration enabled, volumes provisioned using the existing in-tree storage class are automatically migrated to use the AWS EBS CSI storage class. For more information, see xref:persistent-storage-csi-migration.adoc#persistent-storage-csi-migration[CSI automatic migration feature].
```

Comment:
s/the AWS EBS CSI storage class/the AWS EBS CSI driver/ ?
After the CSI migration is enabled, the response for the requests to AWS EBS volumes are processed by the CSI driver, but the storageclass name field of PV and PVC is not changed.